.github/workflows/ci-docs-autogen.yml

```diff
@@ -18,11 +21,12 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Generate and upload docs
-        uses: morpho-labs/foundry-docs-aws@v1
+      - name: Generate & upload forge docs
+        uses: morpho-labs/foundry-docs-aws@v1.1.1
         with:
           aws-s3-bucket: ${{ secrets.AWS_S3_BUCKET }}
           aws-cloudfront-distribution-id: ${{ secrets.AWS_CLOUDFRONT_DISTRIBUTION_ID }}
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          role-session-name: ${{ vars.AWS_SESSION_NAME }}
```

Comment:
What is `vars`?

Reply:
non secret secrets
![image](https://github.com/morpho-org/morpho-v1/assets/61523188/946209be-c23f-4abf-9e3c-f452a7151663)